src/content/docs/nrql/get-started/introduction-nrql-new-relics-query-language.mdx

```diff
@@ -208,6 +208,113 @@ You can also use NRQL with our [NerdGraph API](/docs/apis/graphql-api/tutorials/
 NRQL is one of several ways to query New Relic data. For more on all query options, see [Query your data](/docs/using-new-relic/data/understand-data/query-new-relic-data).
 </Callout>
 
+## How is NRQL structured? [#nrql-rules]
+
+If you're already familiar with SQL queries, you'll be happy to know that NRQL has a lot of similarities. Here's a quick breakdown of the structure of a NRQL query:
```

Comment:
"...familiar with writing SQL queries..."

Reply:
Is it worth mentioning that the underlined text in the codeblock are links to pages on that syntax? I had never realized before today that those were clickable links!

Reply:
First change made, second one I think it might be, but I'm not 100% sure. I will make a note to revisit this in the future after I've mulled it over.